File: client-java/controller-api/src/main/java/org/evomaster/clientJava/controllerApi/dto/database/operations/InsertionEntryDto.java
package org.evomaster.clientJava.controllerApi.dto.database.operations;

public class InsertionEntryDto {

    public String variableName;

    public String printableValue;

    /**
     * If non null, then printableValue should be null
     */
    public Integer foreignKeyToPreviouslyGeneratedRow;
}

File: client-dotnet/src/EvoMaster.Controller.Api/dto/database/operations/InsertionEntryDto.cs
namespace EvoMaster.Controller.Api {
    public class InsertionEntryDto {
        public string VariableName { get; set; }

        public string PrintableValue { get; set; }

        /**
         * If non null, then printableValue should be null.
         * This should be an id of an InsertionDto previously
         * executed
         */
        public long? ForeignKeyToPreviouslyGeneratedRow { get; set; }
    }
}